### PR TITLE
Feature/user login

### PR DESCRIPTION
**Pull request by cmanamela**
Setup actions and reducers for the login page

### CODE REVIEW

File: src/Components/Login/LoginActionTypes.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const 
+    CLEAR_LOGIN_DETAILS = 'CLEAR_LOGIN_DETAILS',
+    LOGIN_PASSWORD_CHANGE = 'LOGIN_PASSWORD_CHANGE',
+    LOGIN_CELLNO_CHANGE = 'LOGIN_CELLNO_CHANGE',
+    LOGIN_SUBMIT_BEGIN = 'LOGIN_SUBMIT_BIGIN',
+    LOGIN_SUCCESS = 'LOGIN_SUCCESS',
+    LOGIN_FAILURE = 'LOGIN_FAILURE',
+    NEDBANK_MONEY_LOGIN_SUBMIT_BEGIN = 'NEDBANK_MONEY_LOGIN_SUBMIT',
+    NEDBANK_MONEY_LOGIN_SUCCESS = 'NEDBANK_MONEY_LOGIN_SUCCESS',
+    NEDBANK_MONEY_LOGIN_FAILURE = 'NEDBANK_MONEY_LOGIN_FAILURE';
```
Comment (GeromeSchutte) [inline src/Components/Login/LoginActionTypes.js]: please keep all your action prefixes the same

File: src/Components/Login/LoginComponent.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+class LoginComponent extends Component {
+    static navigationOptions = {
+        title: 'Login'
+    };
+    constructor(props) {
+        super(props);
+
+        this.passwordRef = React.createRef();
+        this.cellNoRef = React.createRef();
```
Comment (GeromeSchutte) [inline src/Components/Login/LoginComponent.js]: you do not need this ref if nothing is going to use it

File: src/Components/Login/LoginReducer.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        return {
+            ...state,
+            loading: action.loginState.loading,
+            jwt: action.loginState.jwt
+        };
+    case Types.LOGIN_FAILURE:
+        return {
+            ...state,
+            loading: action.loginState.loading
+        };
```
Comment (GeromeSchutte) [inline src/Components/Login/LoginReducer.js]: everything before this line can return {...state, ...action.loginState}

Comment (cmanamela) [reply src/Components/Login/LoginReducer.js]: Fixed. Please have a look I am not sure if I understand this.

**Pull request merged**